### PR TITLE
[C-1729] Add profile/collection card skeletons

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Adds profile/collection card skeletons for all card lists in the app (profile, search, explore, favorites)
- Refactors card types
- Improves loading states
- Renames and consolidates card-lists and cards
- refactors TabInfo to use new style system and core components

### CODE REVIEW

File: packages/mobile/src/components/card/Card.tsx [hunk 1]
```diff
@@ -51,28 +51,28 @@ const useStyles = makeStyles(({ palette, typography, spacing }) => ({
...
+type ProfileCardProps = {
+  type: 'user'
   user: User
 }
 
+type CollectionCardProps = {
+  type: 'collection'
+}
+
+export type CardProps = BaseCardProps & (ProfileCardProps | CollectionCardProps)
```
Comment (sliptype) [inline packages/mobile/src/components/card/Card.tsx:72]: Nice

File: packages/mobile/src/components/skeleton/Skeleton.tsx [hunk 2]
```diff
@@ -87,4 +87,6 @@ export const StaticSkeleton = (props: StaticSkeletonProps) => {
   return <Skeleton noShimmer {...props} />
 }
 
+export { Skeleton }
```
Comment (sliptype) [inline packages/mobile/src/components/skeleton/Skeleton.tsx:90]: Were there too many references to skeleton to replace the default export?

**Review by sliptype — APPROVED** [review]
woooo great changes

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-1729-add-playlist-profile-skeletons

File: packages/mobile/src/components/skeleton/Skeleton.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/components/skeleton/Skeleton.tsx:90]: yeah was just a stop gap for now, but yeah should go in and remove default exports eventually

**Pull request merged**